packages/core/useWebWorker/index.ts

```diff
@@ -55,5 +55,6 @@ export function useWebWorker(
     data,
     post,
     terminate,
+    worker: worker!,
```

Comment:
The problem here is that the worker is a variable and might changes during the lifecyle. Better  to make it a shallowRef internally and expose the ref.

Reply:
Ah yeah I started doing that at first but had some issues with the typing, I'll try to update it to use shallowRef again